code/datums/mood_events/generic_negative_events.dm

```diff
@@ -348,10 +348,15 @@
 	timeout = 5 MINUTES
 
 /datum/mood_event/soda_spill
-	description = span_warning("Cool! That's fine, I wanted to wear that soda, not drink it...\n")
+	description = "<span class='warning'>Cool! That's fine, I wanted to wear that soda, not drink it...</span>\n"
```

Comment:
Why?

Reply:
@JohnFulpWillard said the span macros do not work correctly with the descriptions of mood events

Reply:
Well are they actually right?

Reply:
that doesn't make sense, outside the \n bit

Reply:
Very much agree, you should test if they're right, because I'm almost certain they're not.

The `\n` outside of the span doesn't even matter.

Reply:
they work fine, you just can't do 
"[span_warning("Based")] Other text goes here [span_notice("Okay")]"
Because this doesn't count as a constant expression due to byond. Only reason the rest of the mood events weren't done is because they were a pain to regex so it is pretty inconsistent for some to be span defines and for some to not be.